### ISSUE TITLE
Galacticraft crashing when changing key bindings

### ISSUE THREAD

**Issue by AngryCarrot789** [xJon/Tekkit-2#1]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: 

```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// Daisy, daisy...

Time: 2022-08-27 17:43:30 BST
Description: Initializing game

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from BuildCraft Lib (buildcraftlib)
Caused by: java.lang.Error: Failed to create a page link for galacticraftcore:landing_pad class micdoodle8.mods.galacticraft.core.items.ItemBlockLandingPad ({id:"galacticraftcore:landing_pad",Count:1b,Damage:0s})
    at buildcraft.lib.client.guide.entry.PageEntryItemStack.iterateAllDefault(PageEntryItemStack.java:114)
    at buildcraft.lib.client.guide.GuideManager.generateContentsPage(GuideManager.java:337)
    at buildcraft.lib.client.guide.GuideManager.reload0(GuideManager.java:203)
    at buildcraft.lib.client.guide.GuideManager.reload(GuideManager.java:132)
    at buildcraft.lib.client.guide.GuideManager.onResourceManagerReload(GuideManager.java:119)
    at net.minecraft.client.resources.SimpleReloadableResourceManager.registerReloadListener(SimpleReloadableResourceManager.java:121)
    at buildcraft.lib.BCLibProxy$ClientProxy.fmlPostInit(BCLibProxy.java:173)
    at buildcraft.lib.BCLib.postInit(BCLib.java:142)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
    at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
    at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:754)
    at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:336)
    at net.minecraft.client.Minecraft.init(Minecraft.java:535)
    at net.minecraft.client.Minecraft.run(Minecraft.java:4601)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.IndexOutOfBoundsException
    at java.nio.Buffer.checkIndex(Buffer.java:540)
    at java.nio.DirectByteBuffer.get(DirectByteBuffer.java:253)
    at org.lwjgl.input.Keyboard.isKeyDown(Keyboard.java:407)
    at micdoodle8.mods.galacticraft.core.items.ItemBlockDesc.addInformation(ItemBlockDesc.java:62)
    at net.minecraft.item.ItemStack.getTooltip(ItemStack.java:707)
    at buildcraft.lib.misc.GuiUtil.getUnFormattedTooltip(GuiUtil.java:449)
    at buildcraft.lib.client.guide.parts.contents.PageLinkItemStack.getTooltip(PageLinkItemStack.java:43)
    at buildcraft.lib.client.guide.parts.contents.PageLinkItemStack.create(PageLinkItemStack.java:29)
    at buildcraft.lib.client.guide.entry.PageEntryItemStack.iterateAllDefault(PageEntryItemStack.java:110)
    ... 47 more
```

I added a galacticraft item into my inventory (one of the solar panel items), using NEI (in cheat mode), and it worked fine. But when i clicked the item in my hotbar (in creative mode), it crashed my game. And now when i try to launch the game, i get the above crash

I also changed my sprint key from CTRL to Shift, and i changed my sneak key from Shift to Mouse5

Edit:
I deleted the file `%appdata%\.technic\modpacks\tekkit-2\options.txt` and it seems to have fixed the crash on launch

Edit again:
Seems like it's caused by using mouse buttons in the controls menu, specifically Mouse5 as sneak maybe

**Comment by xJon** [comment]
Caused by Galacticraft, and just patched by @ACGaming over https://github.com/TeamGalacticraft/Galacticraft-Legacy/pull/89. Thank you nonetheless for the report!

**Comment by AngryCarrot789** [comment]
Sweet :)

**Comment by xJon** [comment]
Will keep this open until the fix gets into the modpack :)

**Comment by xJon** [comment]
Fixed in `1.0.1` thanks to @ACGaming :)
